package.json

```diff
@@ -41,8 +41,10 @@
     "dotenv": "^7.0.0",
     "jest": "^24.7.1",
     "jsonwebtoken": "^8.5.1",
+    "lodash": "^4.17.11",
```

Comment:
Why do you need lodash?

Reply:
I am dealing with object and array so many times. Without Lodash, I would end up writing so many custom functions and repeating myself.